mockito2/src/test/java/com/baeldung/mockito/java8/LazyVerificationTest.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.mockito.java8;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import java.util.List;
+
+import org.junit.Test;
+import org.mockito.exceptions.base.MockitoAssertionError;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.VerificationCollector;
+
+public class LazyVerificationTest {
+
+    @Test
+    public void whenLazilyVerified_thenThrowsMultipleExceptions() {
+        VerificationCollector collector = MockitoJUnit.collector()
+            .assertLazily();
+
+        List mockList = mock(List.class);
+        verify(mockList).add("one");
+        verify(mockList).clear();
+
+        try {
```

Comment:
Let's add the exception to the method signature and remove the try-catch

Reply:
Actually, the try-catch block there is to assert that both verification failures are reported when lazy verification is used, and is kind of part of the test. The try method is supposed to fail. I updated method name to make this intent clearer.